src/client/game.ts

```diff
@@ -292,7 +325,11 @@ export default class ClientGame extends GameLoop {
     this.player_list[id] = character;
     this.stage.addChild(character);
     character.show(character.animationStates.down);
-    character.shot_line = this.add_shot_line({ x: x, y: y }, { x: 0, y: 0 });
+    character.shot_line = this.add_shot_line(
+      weapon,
+      { x: x, y: y },
```

Comment:
```suggestion
      { x, y },
```
Jag tror man kan skriva så där